src/content/github.js

```diff
@@ -81,12 +81,29 @@ export default class GitHub {
         continue;
       }
 
-      const prettierButton = renderButton(parentNode, {
+      const options = {
         append: true,
         classes: ["prettier-btn"],
-        refNode: button.innerText === BUTTONS.SUBMIT_NEW_ISSUE ? button : null,
+        refNode: null,
         style: { "margin-right": "4px" }
-      });
+      };
+
+      // These two buttons have a unique DOM structure, so we need
```

Comment:
Trying to simplify and document why we're doing the things we're doing for ease of maintainability. The code has been pretty opaque up to this point.